Moblin/View/Stream/StreamOverlayView.swift

```diff
@@ -2,14 +2,16 @@ import SwiftUI
 
 struct ChatWarning: View {
     var message: String
+    var icon: String
+    var iconColor: Color;
```

Comment:
This struct is called ChatWarning because it should warnings. We can rename it to ChatInfo or something like that.

Reply:
renamed to ChatInfo. I think this will be useful in the future and to duplicate code isn't helpful